### PR TITLE
Register and tag Tokenize.jl [https://github.com/KristofferC/Tokenize.jl]

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

File: Tokenize/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4
```
Comment (tkelman) [inline Tokenize/versions/0.0.1/requires]: should be tested on 0.4 if supported there

**Comment by tkelman** [comment]
Anything takrn from JuliaParser should be acknowledged in the top-level license. If JuliaParser does this, why not improve  the existing package?

**Comment by KristofferC** [comment]
Added juliaparser license and removed support for 0.4.

This isn't an improvement over JuliaParser, it does a different thing. Please read the README in Tokenize.jl and compare what JuliaParser does if you are interested.

**Comment by KristofferC** [comment]
Gonna try attobot.

**Pull request closed without merging**